src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -238,4 +245,43 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
+
+  revokeToken() {
+    let dialogRef = this.dialog.open(RevokeTokenDialogComponent, {
+      width: bootstrap4largeModalSize,
+    });
+    dialogRef.afterClosed().subscribe((result) => {
+      if (result) {
+        this.deleteToken(TokenSource.DOCKSTORE).subscribe(
+          () => {
+            this.revokeTokenSuccess();
+          },
+          () => {
+            this.revokeTokenFailure();
+          }
+        );
+      }
+    });
+  }
+
+  /**
+   * What happens when revoking the Dockstore Token has succeeded.
+   *
+   * @private
+   * @memberof RevokeTokenDialogComponent
+   */
+  private revokeTokenSuccess(): void {
+    this.logoutService.logout();
+    this.matSnackBar.open('Revoking Dockstore token succeeded', 'Dismiss');
```

Comment:
Sorry, I missed this before, but you should use the AlertService to display these kinds of messages. While you're code does what the AlertService currently does, it duplicates it. If it's not duplicated, and we want to change the way alerts look, we would only have to change it one place.